### ISSUE TITLE
Console error related to Codeium requests

### ISSUE THREAD

**Issue by lihebi** [codepod-io/codepod#360]
This error is printed a lot of times. It appears to me that multiple Codeium requests are sent and some throw errors. Related to #328 

<img width="708" alt="Screenshot 2023-07-07 at 1 10 25 PM" src="https://github.com/codepod-io/codepod/assets/4576201/ac119d26-80e7-4042-bd0a-a880e6bb5251">

**Comment by li-xin-yi** [comment]
Yeah, I also tried to fix them a week ago. But I have no idea, the official Codeium chrome plug-in shows the same bug when you interrupt the process of auto-completion (e.g., reject the completion fast and type other characters). I guess the issue roots from the server-end, let me investigate their source code when I have time.
